ota-plus-web/app/reactapp/src/containers/Terms.jsx

```diff
@@ -25,13 +46,17 @@ export default class Terms extends Component {
                         <p>I AGREE TO OUR <a href="#">SERVICE TERMS</a> AND <Link to="/policy">PRIVACY POLICY.</Link></p>
```

Comment:
This should link to the terms about to be accepted.

Reply:
@taheris But we have a new design where these links are external links to HERE pages